src/epcpm/anomaliestoc.py

```diff
@@ -0,0 +1,104 @@
+import attr
+import pathlib
+
+import epyqlib.pm.parametermodel
+import epyqlib.utils.general
+
+import epcpm.c
+import epcpm.pm_helper
+import epcpm.anomalymodel
+
+
+builders = epyqlib.utils.general.TypeMap()
+
+
+def export(
+    anomaly_model: epyqlib.attrsmodel.Model,
+    parameters_model: epyqlib.attrsmodel.Model,
+    h_path: pathlib.Path,
+) -> None:
+
+    builder = epcpm.anomaliestoc.builders.wrap(
+        wrapped=anomaly_model.root,
+        parameter_uuid_finder=parameters_model.node_from_uuid,
+    )
+
+    data = builder.gen()
+
+    epcpm.c.render(
+        source=h_path.with_suffix(f"{h_path.suffix}_pm"),
+        destination=h_path,
+        context={"anomaly_tables": data},
+    )
+
+
+@builders(epcpm.anomalymodel.Root)
+@attr.s
+class Root:
+    wrapped = attr.ib()
+    parameter_uuid_finder = attr.ib()
+
+    def gen(self):
+
+        items = []
+        for anomaly_table in self.wrapped.children:
+            item = builders.wrap(
+                wrapped=anomaly_table,
+                parameter_uuid_finder=self.parameter_uuid_finder,
+            ).gen()
+            items.append(item)
+
+        return items
+
+
+@builders(epcpm.anomalymodel.AnomalyTable)
+@attr.s
+class AnomalyTable:
```

Comment:
Would it be better for the vocabulary to be "AnomalyGroup" instead of "AnomalyTable"?

Reply:
On firmware repos m-tcu and rm-dc, they are consistently named "Tables", so I think it would be better to keep that name to avoid confusion.